adobe/spectrum-web-components
# fix(picker): correct label application for screen readers

## Description
At some point Chrome apparently decided that you can use IDREFs to elements who's parent has `aria-label`. This updates the labelling patter for Picker to account for this. I didn't do a larger refactor of Field Label application into custom elements, because I'd like to believe that we can wait for up coming browser APIs before we do that. It may be appropriate at a intermediate date to move for a well known property to a well known method and then allow the labelled element to _always_ decide how to apply the label, which would prevent the need to hijack the native DOM method from the outside like this does.

## Related issue(s)
- fixes #3238

## Motivation and context
#a11y

## How has this been tested?

    1. Go [here](https://picker-label--spectrum-web-components.netlify.app/storybook/?path=/story/picker--default) in Chrome in macOS
    2. Turn on Voice Over
    3. See that the Picker is appropriately announced\

## Types of changes

## Checklist
